README.md

```diff
@@ -1,24 +1,91 @@
+| **:open_file_folder: Supporting packages** |
+| &nbsp;&nbsp;[@patternfly/react-docs](https://github.com/patternfly/patternfly-react/tree/master/packages/patternfly-4/react-docs) | Gatsby documentation site for components |
+| &nbsp;&nbsp;[@patternfly/react-integration](https://github.com/patternfly/patternfly-react/tree/master/packages/patternfly-4/react-integration) | Cypress integration tests |
+| **:family: Extension packages** *community supported* | 
+| &nbsp;&nbsp;[@patternfly/react-catalog-view-extension](https://github.com/patternfly/patternfly-react/tree/master/packages/patternfly-4/react-catalog-view-extension) | Catalog view extension |
+| &nbsp;&nbsp;[@patternfly/react-virtualized-extension](https://github.com/patternfly/patternfly-react/tree/master/packages/patternfly-4/react-virtualized-extension) | Table and list row virtualization extension |
+| **:x: Deprecated packages** |
+| &nbsp;&nbsp;[@patternfly/react-inline-edit-extension](https://github.com/patternfly/patternfly-react/tree/master/packages/patternfly-4/react-inline-edit-extension) | Table inline edit extension |
+| &nbsp;&nbsp;[@patternfly/react-styled-system](https://github.com/patternfly/patternfly-react/tree/master/packages/patternfly-4/react-styled-system) | Styled system component prototype |
+| &nbsp;&nbsp;[@patternfly/react-codemods](https://github.com/patternfly/patternfly-react/tree/master/packages/react-codemods) | Codemods for upgrading PatternFly 3 components |
 
-This project is governed by the Contributor Covenant version 1.4. All contributors and participants agree to abide by its terms. To report violations, send an email to patternfly@redhat.com.
+### Setup
 
-### Contributing Guide
+Before you begin, check out this [overview of PatternFly](http://patternfly.org/v4/get-started/about) to get familiar with the basic elements of the design system.
+
+#### Install a package manager
+Install a package manager before using the PatternFly libraries.
+
+* [Use npm](https://nodejs.org/en/download)
+* [Use Yarn](https://yarnpkg.com/en/docs/getting-started)
+
+
+#### Start with PatternFly React Seed
+If you want to start with your existing project, skip to [Install and configure PatternFly React](#install-and-configure-patternfly-react).
+
+[PatternFly React seed](https://github.com/patternfly/patternfly-react-seed) is an open source build scaffolding utility. It gives developers a jump-start when creating new projects that will use PatternFly.
+
+
+1. **Fork the project**
+
+  * TypeScript: [Fork and clone the PatternFly React Seed project](https://github.com/patternfly/patternfly-react-seed).
+  * JavaScript: [Fork and clone the JavaScript branch of the PatternFly React Seed project](https://github.com/patternfly/patternfly-react-seed/tree/javascript).
+
+  For more information about how to fork and clone a repo, [view GitHub Help](https://help.github.com/articles/fork-a-repo/).
```

Comment:
Should this be indented?  When I look at the rich text it is not aligned under the "1."  I don't know if it should be.

Reply:
Updated alignment earlier.